share/wake/lib/system/io.wake

```diff
@@ -75,9 +75,8 @@ export def installAs (dest: String) (file: Path): Result Path Error =
     makeExecPlan cmd inputs
     | setPlanLabel "installAs: {dest} {file.getPathName}"
     | setPlanEnvironment Nil
-    | setPlanLocalOnly True
     | setPlanFnOutputs foutputs
-    | runJob
+    | runJobWith localRunner
```

Comment:
I'm sure I'm just missing something, but why does this have to be `localRunner`?  The `file` is already a Path so it shouldn't have any issues being sandboxed, and the parent `destDir` is created with `mkdir` and already in the visible list.

Side note, I think the `mkdir "{dest}/.."` might be buggy if anyone tries to `installAs` something directly into the workspace root?

Reply:
I blindly converted this, but looking at it now I see what you mean though. I believe the rational was that some 'basic' level operations could be expected to behave well without the overhead of starting a sandbox, such as in this case calling `cp`.
Additionally it uses the `which` function which implies that we're expecting to resolve the path to `cp` from the host's `$PATH` instead of within a sandbox